recipes/r-vioplot/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# 'Autobrew' is being used by more and more packages these days
+# to grab static libraries from Homebrew bottles. These bottles
+# are fetched via Homebrew's --force-bottle option which grabs
+# a bottle for the build machine which may not be macOS 10.9.
+# Also, we want to use conda packages (and shared libraries) for
+# these 'system' dependencies. See:
+# https://github.com/jeroen/autobrew/issues/3
+export DISABLE_AUTOBREW=1
```

Comment:
@rasmusagren Did you add this section about Autobrew yourself? If yes, why is it necessary for building r-vioplot (which appears to me to be a pure R package)? If no, what version of conda build did you use that added this text?

Reply:
Thanks for a rapid reply! This was added automatically when using `conda skeleton cran`. I'm using conda-build 3.4.2 from conda-forge.

Reply:
Sorry about that. conda-forge has updated the conda-build recipe to version 3.4.2, but it is still using conda-build 2 for creating recipes and building packages. Confusing I know. For the foreseeable future, you should install conda-build 2 with `conda install -c conda-forge conda-build=2*`. At some point we'll migrate to conda-build 3, so I'd recommend checking the docs regularly.